cypress/e2e/marc-authority/marc-bibliographic-unlink-fields.cy.js

```diff
@@ -0,0 +1,171 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import { DevTeams, TestTypes, Permissions } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import Users from '../../support/fragments/users/users';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+
+describe('MARC -> MARC Bibliographic', () => {
+  const testData = {
+    tag100: '100',
+    tag010: '010',
+    tag700: '700',
+    authority100FieldValue: 'C365598 Chin, Staceyann, 1972-',
+    authority010FieldValue: 'n2008052404',
+    authority700FieldValue: 'C365598 Woodson, Jacqueline',
+    successMsg:
+      'This record has successfully saved and is in process. Changes may not appear immediately.',
+    accordion: 'Contributor',
+  };
+  let createdInstanceID;
+  const marcFiles = [
+    {
+      marc: 'marcFileForC365598_1.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcFileForC365598_2.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcFileForC365598_3.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      numOfRecords: 1,
+    },
+  ];
+  const linkedField = {
+    rowIndex: 11,
+    tag: '100',
+    secondBox: '1',
+    thirdBox: '\\',
+    content: '$a C365598 Chin, Staceyann, $d 1972-',
+    eSubfield: '$e Author $e Narrator',
+    zeroSubfield: '$0 id.loc.gov/authorities/names/n2008052404365598',
+    seventhBox: '$1 http://viaf.org/viaf/24074052',
+  };
+  const unlinkedField = {
+    rowIndex: 11,
+    tag: '100',
+    indicator0: '1',
+    indicator1: '\\',
+    content:
+      '$a C365598 Chin, Staceyann, $d 1972- $e Author $e Narrator $0 id.loc.gov/authorities/names/n2008052404365598 $1 http://viaf.org/viaf/24074052',
+  };
+  const contributors = {
+    firstName: 'C365598 Chin, Staceyann, 1972-',
+    secondName: 'Woodson, Jacqueline',
+  };
+  const createdAuthorityIDs = [];
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
```

Comment:
Permissions set is not correct, should be:
```JS
      Permissions.inventoryAll.gui,
      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
      Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
      Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
```

Reply:
resolved